### PR TITLE
Added Safari cheat sheet

### PR DESCRIPTION
**Pull request by nikhilzac95**
**What does your Instant Answer do?**
Displays Safari web browser cheat sheet.

**What problem does your Instant Answer solve (Why is it better than organic links)?**
Shows all the daily use shortcuts used in Safari browser instantly.

**What is the data source for your Instant Answer? (Provide a link if possible)**
http://www.shortcutworld.com/en/mac/Safari_5.html

**Why did you choose this data source?**
Trusted source.

**Are there any other alternative (better) data sources?**
No.

**What are some example queries that trigger this Instant Answer?**
safari cheat sheet, Apple Safari Browser cheat sheet

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
All peoples

screenshot from 2015-09-26 13 19 40

-----
IA Page: https://duck.co/ia/view/safari_cheat_sheet

### CODE REVIEW

**Comment by nikhilzac95** [comment]
#duckdukhackcek

**Comment by nikhilzac95** [comment]
@moollaza check this

**Comment by gautamkrishnar** [comment]
#1611 duckdukhackcek

**Comment by moollaza** [comment]
@nikhilzac95 Hey!

Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

File: share/goodie/cheat_sheets/json/safari.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  "id": "safari_cheat_sheet",
+  "name": "Apple Safari",
+  "description": "Keyboard Shortcuts for Safari Web Browser",
+  "template_type": "keyboard",
+  "metadata": {
+    "sourceName": "Shortcutworld",
+    "sourceURL": "http://www.shortcutworld.com/en/mac/Safari_5.html"
+  },
+  "aliases": [
+    "Apple Safari Browser"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/safari.json]: I would add "safari browser", and "apple safari"

File: share/goodie/cheat_sheets/json/safari.json [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  ],
+  "section_order": [
+    "Manage Tabs and Windows",
+    "Safari Features and Bookmarks"
+  ],
+  "sections": {
+    "Safari Features and Bookmarks": [
+      {
+        "key": "[command] [option] [1]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/safari.json]: Let's capitalize `[Command]` and `[Option]`

Comment (nikhilzac95) [inline share/goodie/cheat_sheets/json/safari.json]: sure.. Will do :)

Sent from my ASUS

-------- Original Message --------

**Comment by gautamkrishnar** [comment]
@moollaza :+1:

**Comment by nikhilzac95** [comment]
@moollaza Again checks have failed!
please tell me what the issue is so that I can fix it..
#1611

**Comment by moollaza** [comment]
@nikhilzac95 @gautamkrishnar sorry for the delay. You need to lowercase your alias `"apple safari browser"` and then the tests will pass

**Comment by nikhilzac95** [comment]
surely.. will do now..

Sent from my ASUS

-------- Original Message --------

**Comment by gautamkrishnar** [comment]
:+1:

**Comment by codenirvana** [comment]
@nikhilzac95 @moollaza We need to close this PR.
#1721 already merged!

**Comment by gautamkrishnar** [comment]
@moollaza its okay. I think @nikhilzac95 will no longer be able to work on this.

**Pull request closed without merging**